pkg/types/cciptypes/commitstore.go

```diff
@@ -0,0 +1,81 @@
+package cciptypes
```

Comment:
let's make this `ccip` rather than `cciptypes` to match the packages here

Reply:
i'd prefer the existing pattern for a few reasons: it removes the from stutter `...types/cciptypes`, the call sites can alias to `cciptypes` upon import, it's maintains consistency. finally, the golang idiom here is `types/ccip` for the first two reasons.